tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -1679,13 +1673,31 @@ class ParameterizedOpConverterTest
       public ::testing::WithParamInterface<
           std::tuple<TrtTestMode, DataType, TrtPrecisionMode>> {};
 
-// Build and the converted network. Check output tensor shape. Check if the
-// output values agree with the output_vec argument.
+// Instantiate parameter combinations to test. For debugging purposes it might
+// make sense to run over all possible combinations, but normally a subset of
+// them would be sufficient:
+// - All valid options to TrtTestMode (implicit, explicit, dynamic shape)
+// - DataType: is the TF data type of the input tensors. This usually only
+//   influences the data type added by Converter::AddInputTensor. That function
+//   is tested separately for valid combinations of input data types, therefore
+//   for most of the converter its is sufficient to test for DT_FLOAT.
+// - TrtPrecisionMode: valid options are FP32, FP16 and INT8. This influences
+//   how TRT handles the precision inside the TRT network, but should not matter
+//   for the TF -> TRT conversion. Therefore it should be sufficient to test
+//   for FP32.
```

Comment:
It would be nice if we can find a way to test all the combination for one op.

Reply:
Can we change the description for this class to explicit stay that it is for testing ops with combination of (implicit, explicit, dynamic shape) x (DT_FLOAT) X (TrtPrecisionMode::FP32)?

Reply:
Yes, I have changed the description.